demos/benchmark/embeddings/benchmark_embeddings.py

```diff
@@ -37,19 +37,24 @@
 parser = argparse.ArgumentParser(description='Run benchmark for embeddings endpoints', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 parser.add_argument('--dataset', required=False, default='Cohere/wikipedia-22-12-simple-embeddings', help='Dataset for load generation from HF or a keyword "synthetic"', dest='dataset')
 parser.add_argument('--synthetic_length', required=False, default=510, type=int, help='Length of the synthetic dataset', dest='length')
-parser.add_argument('--api_url', required=False, default='http://localhost:8000/v3/embeddings', help='API URL for embeddings endpoint', dest='api_url')
+parser.add_argument('--api_url', required=False, help='API URL for embeddings endpoint', dest='api_url')
```

Comment:
If there's no default then maybe this parameter should be required?

Reply:
the default is set depending on the serving backend